src/components/com_tjcertificate/site/views/certificates/tmpl/my.php

```diff
@@ -106,8 +123,12 @@
 										echo '-';
 									}
 									?></td>
-								<td><?php echo $this->escape($item->comment); ?></td>
-								<td><?php echo (int) $item->id; ?></td>
+								<td>
+									<?php 
+										$client = explode('.', $item->client); 
+										echo ucfirst($client[1]);
+									?>
```

Comment:
You can use lang constant here as you have used in the filters

Reply:
done